app/queues/certificate/order-completer-worker.server.ts

```diff
@@ -74,7 +75,15 @@ export const orderCompleterWorker = new Worker<CertificateJobData>(
       ({ privateKey, certificate, validFrom, validTo } = await letsEncrypt.completeOrder());
     } catch (e) {
       logger.error('failed to finalize certificate', e);
-
+      // send error email notification:
+      logger.debug(
```

Comment:
Probably we should do the failure case as part of #511, since there are so many ways that this flow can fail.  It's hard to do it in the worker, since we retry jobs.